spec/vast_trackers.spec.js

```diff
@@ -13,13 +13,15 @@ describe('VASTTracker', function() {
     let spyTrack;
     let adTrackingUrls;
     let ad;
+    let isCustomCode = false;
```

Comment:
This variable doesn't need to be global as it's already defined in the unit test that you have added